sklearn/datasets/tudarmstadt.py

```diff
@@ -0,0 +1,126 @@
+"""TU Darmstadt dataset.
```

Comment:
maybe I'll just add this to the example, and not put it in the datasets folder. This is not really a widely used dataset. and once it is in the public api, it is hard to get rid of.